src/main/java/bio/terra/app/utils/TimUtils.java

```diff
@@ -50,19 +48,14 @@ public static String encode(String s) {
    * Encode the AS columns of a SQL query into corresponding TIM property names
    *
    * @param sql the SQL string containing AS columns to encode
-   * @param columnReplacements a map which stores column names as keys and TIM property names as
-   *     values
    * @return a modified SQL string containing encoded TIM property names
    */
-  public static String encodeSqlColumns(String sql, Map<String, String> columnReplacements) {
-    Pattern regex = Pattern.compile("( [aA][sS] )(\\w+)");
+  public static String encodeSqlColumns(String sql) {
+    Pattern regex = Pattern.compile("( [aA][sS] )\\[([\\w.:]+)\\]");
```

Comment:
I was thinking that we don't need the `[]`s in SQL to set apart the TIM names, since the SQL is easier to parse than the ES query. I was thinking users could write `SELECT x AS  TerraCore:something`. Using a regex like
```
    Pattern regex = Pattern.compile("( [aA][sS] )([\\w:.]+)");
```
would work.

Also can this be written using a single capture group, or do we need to capture the `AS` so it's not lost when we replace the tim name?

Reply:
I was thinking the brackets would be nicely symmetric to the ES query, but I am not super attached to the choice. We need to keep both capture groups so they aren't lost.

Reply:
OK. I'm thinking that at some point in the future we could have an API where both the SQL and ES query strings are generated from a collection of inputs instead of a single string, and at that point we could change to not requiring the extra brackets. For now it seems fine.